docker/README.md

```diff
@@ -108,21 +108,46 @@ Note: please update this README with the exact wording of the error if you run i
 
 `OSError: [Errno 12] Cannot allocate memory:` could occur in conjunction with `/openlibrary/openlibrary/core` or any number of files. Simply try increasing free RAM or increasing swap/page file/virtual memory for your operating system.
 
-
 #### "No module named 'infogami'"
 
 The following should populate the target of the `infogami` symbolic link (i.e. `vendor/infogami/`):
 ```
 cd path/to/your/cloned/openlibrary
 git submodule init; git submodule sync; git submodule update
 ```
-
 Windows users may need to see [Fix line endings, symlinks, and git submodules](https://github.com/internetarchive/openlibrary/wiki/Git-Cheat-Sheet#fix-line-endings-symlinks-and-git-submodules-only-for-windows-users-not-using-a-linux-vm).
 
 #### "no configuration file provided: not found" when running `docker compose <command>`
 
 Ensure you're running `docker compose` commands from within the `local-openlibrary-dev-directory`.
 
+### ConnectionError: HTTPConnectionPool(host='solr', port=8983)
+The full error is something like (line breaks added):
+```
+/openlibrary/openlibrary/templates/home/index.html: error in processing
+ template: ConnectionError: HTTPConnectionPool(host='solr', port=8983):
+Max retries exceeded with url: /solr/openlibrary/select (Caused by
+NameResolutionError("<urllib3.connection.HTTPConnection object at
+0x77a95c4e7f90>: Failed to resolve 'solr' ([Errno -2] Name or service
+not known)")) (falling back to default template)
+```
+First, verify the `solr` container is running (e.g. `docker ps | grep solr`, and then look for something like `openlibrary-solr-1` that isn't `solr-updater`.) Verify too that you can also connect to solr at http://localhost:8983/solr/#/. If you can't, try `docker compose up -d solr`, and verify that it is running and you can connect to it.
+
+If the `solr` container is running and the error persists, one cause seems to be that the containers become disconnected from `openlibrary_webnet` (though conceivably this could happen with `openlibrary_dbnet` too). `openlibrary-web-1`/`web` should be connected to both `openlibrary_webnet` and `openlibrary_dbnet`, but instead only one is connected. E.g.:
+```sh
+❯ docker container inspect --format '{{.NetworkSettings.Networks}}' openlibrary-web-1
+map[openlibrary_dbnet:0xc00037c1c0]
+```
+A fix seems to be:
+```
+❯ docker network connect openlibrary_webnet openlibrary-web-1
+❯ docker container inspect --format '{{.NetworkSettings.Networks}}' openlibrary-web-1
```

Comment:
```suggestion
docker container inspect --format '{{.NetworkSettings.Networks}}' openlibrary-web-1
# output: map[openlibrary_dbnet:0xc00037c1c0]
```
A fix seems to be:
```
docker network connect openlibrary_webnet openlibrary-web-1
docker container inspect --format '{{.NetworkSettings.Networks}}' openlibrary-web-1
```

I'd prefer not thing those icons at the front, a little harder to copy/paste.

Reply:
Whoops. I realized that change removed the fix. I updated it again, and removed the prompt.

There is some part of me that wants to discourage blindly copying and pasting and to instead encourage people to type the commands in the hope they will engage with them more and perhaps understand a bit more about what's actually going on, taking it line by line, but that's likely a fantasy.